src/lib/functions/registry.mjs

```diff
@@ -191,6 +196,7 @@ export class FunctionsRegistry {
         buildRustSource: env.NETLIFY_EXPERIMENTAL_BUILD_RUST_SOURCE === 'true',
         project_deploy_configuration_api_use_per_function_configuration_files: true,
       },
+      configFileDirectories: [join('.netlify', INTERNAL_FUNCTIONS_FOLDER)],
```

Comment:
We should use the `getPathInHome` helper rather than hardcoding `.netlify` here.

Reply:
You mean like what I did now? :)

Reply:
Sorry, I meant `getPathInProject`, which already contains `.netlify`: https://github.com/netlify/cli/blob/main/src/lib/settings.mjs#L35-L38

Reply:
Haha that makes more sense ☺️ thanks!